azure/templates/sandbox-tests.yml

```diff
@@ -3,14 +3,26 @@ steps:
     workingDirectory: $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)
     displayName: Setup sandbox tests
 
+  - ${{ if eq(parameters.environment, 'sandbox') }}:
+    - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
+      parameters:
+        secret_ids:
+          - ptl/azure-devops/personal-demographics/int/APIGEE_APP_ID
+      displayName: Set APIGEE_APP_ID for sandbox env
+  
+  - ${{ if ne(parameters.environment, 'sandbox') }}:
+    - bash: |
+        echo "#vso[task.setvariable variable=APIGEE_APP_ID;issecret=true]"
+      displayName: Set APIGEE_APP_ID for non-sandbox env
```

Comment:
[get-aws-secrets-and-ssm-params.yml](https://github.com/NHSDigital/api-management-utils/blob/334a6a638101cdce7013e89697c7d908dcf376ba/azure/components/get-aws-secrets-and-ssm-params.yml) uses task.setvariable to assign the APIGEE_APP_ID when it exists for the int/prod sandbox env. This is mimicking that behaviour for non-sandbox envs, so that the cli arg `$(APIGEE_APP_ID)` can still be called without erroring out.